api/envoy/config/core/v3/base.proto

```diff
@@ -444,3 +444,15 @@ message ControlPlane {
   // the Envoy is connected to.
   string identifier = 1;
 }
+
+// Configuration of DNS resolver option flags which control the behavior of the DNS resolver.
+message DnsResolverOptions {
+  // Always use TCP queries instead of UDP queries for DNS lookups.
```

Comment:
"Use TCP for all DNS queries instead of the default protocol UDP."

Reply:
Made this change. I had just copied the description from existing without changing it. Always better to make it more apt.